doc/index.md

```diff
@@ -1337,68 +1353,125 @@ On success, returns the response [*headers*](#http.headers) and a [*stream*](#st
 
 ## http.server
 
-This interface is **unstable**.
+The http.server object is used to listen on sockets and respond to all types of HTTP requests. Each client request is threaded and independent of the other client requests. An `http.server` is instantiated by passing a number of options including an `onstream` response handler that processes the HTTP requests and provides a stream mechanism for response. `onstream` can also be used for testing and upgrading a request, with HTTP 1.1 to WebSockets being the notible example. The server can be made non-blocking when used in conjunction with cqueues. 
```

Comment:
Inconsistent about putting "http.server" in backticks.

Spelling: `notible`


> The server can be made non-blocking when used in conjunction with cqueues. 

It *is* non-blocking already: onstream handlers are run in parallel